docs/spec/literal.rst

```diff
@@ -733,18 +733,16 @@ Conditional statements and expressions work as expected:
         return result  # OK
 
 
-Interaction with TypeVars and Generics
-""""""""""""""""""""""""""""""""""""""
+Interaction with Type Variables and Generics
```

Comment:
Out of scope for this PR but I feel this whole section is basically redundant and could be deleted. LiteralString is a type, so it can be used in any place that an arbitrary type can be used. Some of the Literal-related text above is similarly redundant.